docs/UserGuide.md

```diff
@@ -442,7 +443,7 @@ startlesson /c CS2103T
 
 <br>
 
-### Setting number of lessons of a class : `setlesson /c`
+### Setting the number of lessons of a class : `setlesson /c`
```

Comment:
Can you update this in the table of content and its hyperlink too thanks!

Reply:
Okay!